inc/wp-components/classes/class-head.php

```diff
@@ -144,7 +144,11 @@ public function add_tag( $tag, $attributes = [] ) : self {
 	 * @return self
 	 */
 	public function query_has_set() : self {
-		$this->set_title( $this->get_the_head_title() . $this->get_trailing_title() );
+		$title = $this->get_the_head_title();
+		if ( apply_filters( 'wp_components_head_append_trailing_title', '__return_true' ) ) {
```

Comment:
Why not use `true` as the default value?

Reply:
Right, wasn't thinking.